examples/README.md

```diff
@@ -0,0 +1,26 @@
+# Sonic Disc Reader
+A sample Arduino sketch that demonstrates how to use the SonicDisc when attached via I2C.
+
+Primarily, it is shown how to:
+* Initiate SonicDisc measurements
+* Read data
+* Filter data based on variance
+
+# SonDisk_Proc3
+This sketch provides a visual interface for the data produced by SonicDiskReader. Connect an arduino with a SonicDisc to the computer,
+and then open the processing sketch. Use the number keys to select the COM port for the arduino, and the output data from SonicDiskReader will be
+displayed in a ring.
+
+![Screenshot](Proc3.png)  
```

Comment:
Please upload the image on [imgur](imgur.com). :+1:

Reply:
I got rid of the image since I don't have an imgur account and it didn't seem to add that much value.